runtime/fiber.c

```diff
@@ -418,6 +418,61 @@ rewrite_exception_stack(struct stack_info *old_stack,
     fiber_debug_log ("exn_ptr is null");
   }
 }
+
+#ifdef WITH_FRAME_POINTERS
+/* Update absolute base pointers for new stack */
+static void rewrite_frame_pointers(struct stack_info *old_stack,
+    struct stack_info *new_stack)
+{
+  struct frame_walker {
+    struct frame_walker *base_addr;
+    uintnat return_addr;
+  } *frame, *next;
+  ssize_t delta;
+  void *top, **p;
+
+  delta = (char*)Stack_high(new_stack) - (char*)Stack_high(old_stack);
+
+  /* Walk the frame-pointers linked list */
+  for (frame = __builtin_frame_address(0); frame; frame = next) {
```

Comment:
Is it okay to use this GCC builtin (also supported by clang) which returns the current rbp register value?
An alternative would be to implement our own caml_frame_address in amd64 (and other architectures that requires it).

Reply:
At this stage, this should be fine, yes - `--enable-frame-pointers`-mode has always been for gcc/clang on Linux x86_64.